### DISCUSSION TITLE
Script does not work proprely on Debian 12

### DISCUSSION THREAD

**Discussion by GhostDigger** [PR3SIDENT/enshrouded-server#37] [General]
Hello, first thank you for the Linux script. 

There's a problem installing SteamCMD on Debian 12. 

Adding these lines will make it work: 
```
sudo apt update; sudo apt install software-properties-common; sudo apt-add-repository non-free; sudo dpkg --add-architecture i386; sudo apt update
sudo apt install steamcmd
```

(Valve documentation and tested)
Thanks for the work, I don't know if I should post here but I hope to help.

**Comment by stafebak** [comment]
Just wanna thank you for this. I am very amateur in Linux, and was having issues that we're hurting my fragile amateur ego. This saved me. LOL.
